internal/dataplane/kongstate/kongstate_test.go

```diff
@@ -1307,3 +1312,97 @@ func TestFillVaults(t *testing.T) {
 		})
 	}
 }
+
+func TestFillOverridesFailures(t *testing.T) {
```

Comment:
```suggestion
func TestFillOverrides_ServiceFailures(t *testing.T) {
```
How about changing the name to better reflect what is being tested?